src/app/components/media/MediaComponent.js

```diff
@@ -88,6 +146,17 @@ class MediaComponent extends Component {
     }
   }
 
+  setPlayerRect = () => {
+    // update player rect which used to anchor video anno drawer
+    if (this.state.playerRef) {
```

Comment:
See my comments on refs further down. TL;DR should be `this.playerRef.current`

Reply:
Fixed in 5ce99d5fa2fba2c6110cbd4aa70b408a35125d1a